install_config/oab_broker_configuration.adoc

```diff
@@ -0,0 +1,590 @@
+[[install-config-oab-broker-config]]
+= OpenShift Ansible Broker Configuration
+{product-author}
+{product-version}
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+When the
+xref:../architecture/service_catalog/ansible_service_broker.adoc#arch-ansible-service-broker[OpenShift
+Ansible broker] (OAB) is deployed in a cluster, its behavior is largely dictated
+by the broker's configuration file loaded on startup. The broker's configuration
+is stored as a ConfigMap object in the broker's namespace
+(*ansible-service-broker* by default).
+
+.Example OpenShift Ansible Broker Configuration File
+[source,yaml]
```

Comment:
Grabbed this from https://github.com/openshift/ansible-service-broker/blob/master/etc/example-config.yaml.